tests/app/notifications/test_validators.py

```diff
@@ -55,135 +56,66 @@ def enable_redis(notify_api):
         yield
 
 
-@pytest.mark.parametrize('key_type', ['test', 'team', 'normal'])
-def test_check_service_message_limit_in_cache_with_unrestricted_service_is_allowed(
-        key_type,
-        sample_service,
-        mocker):
-    mocker.patch('app.notifications.validators.redis_store.get', side_effect=[
-        None,  # The serialised service
-        1,  # The rolling count
-    ])
-    mocker.patch('app.notifications.validators.redis_store.set')
-    mocker.patch('app.notifications.validators.services_dao')
-    serialised_service = SerialisedService.from_id(sample_service.id)
-
-    check_service_over_daily_message_limit(key_type, serialised_service)
-    app.notifications.validators.redis_store.set.assert_called_once_with(
-        f'service-{serialised_service.id}',
-        ANY,
-        ex=ANY,
-    )
-    assert not app.notifications.validators.services_dao.mock_calls
-
-
-@pytest.mark.parametrize('key_type', ['test', 'team', 'normal'])
+@pytest.mark.parametrize('key_type', ['team', 'normal'])
 def test_check_service_message_limit_in_cache_under_message_limit_passes(
         key_type,
         sample_service,
         mocker):
-    mocker.patch('app.notifications.validators.redis_store.get', side_effect=[
-        None,  # The serialised service
-        1,  # The rolling count
-    ])
-    mocker.patch('app.notifications.validators.redis_store.set')
-    mocker.patch('app.notifications.validators.services_dao')
     serialised_service = SerialisedService.from_id(sample_service.id)
+    mock_get = mocker.patch('app.notifications.validators.redis_store.get', return_value=1)
+    mock_set = mocker.patch('app.notifications.validators.redis_store.set')
     check_service_over_daily_message_limit(key_type, serialised_service)
-    app.notifications.validators.redis_store.set.assert_called_once_with(
-        f'service-{serialised_service.id}',
-        ANY,
-        ex=ANY,
-    )
-    assert not app.notifications.validators.services_dao.mock_calls
+    mock_get.assert_called_once_with(f'{serialised_service.id}-{datetime.utcnow().strftime("%Y-%m-%d")}-count')
+    assert mock_get.called
+    assert not mock_set.called
 
 
 def test_should_not_interact_with_cache_for_test_key(sample_service, mocker):
     mocker.patch('app.notifications.validators.redis_store')
-    mocker.patch('app.notifications.validators.redis_store.get', side_effect=[None])
+    mock_get = mocker.patch('app.notifications.validators.redis_store.get', side_effect=[None])
```

Comment:
Minor: do we need this? We're not calling the `redis_store`, and we've got an assertion for that later on. I'm not sure why we need two assertions (and the extra mock).

Reply:
I think this one got missed, by the way: we're still mocking the Redis get operation twice (and asserting on it twice).